packages/orbit-components/src/Illustration/types.d.ts

```diff
@@ -88,12 +88,8 @@ export type Name =
   | "TransportTaxi"
   | "WomanWithPhone";
 
-// TODO: remove spaceAfter in the next major version
-
-/** spaceAfter is deprecated, use margin instead */
 export interface Props extends Common.Globals, Common.SpaceAfter {
   readonly size?: "extraSmall" | "small" | "medium" | "large" | "display";
   readonly name: Name;
-  readonly margin?: React.CSSProperties["margin"] | Common.ObjectProperty;
   readonly alt?: string;
```

Comment:
Why is this happening here? We are saying spaceAfter is deprecated and to use margin, but margin is being removed. And not related to this PR (?)

Reply:
it's because it's generated from the template